package.json

```diff
@@ -13,10 +13,10 @@
     "stylelint-scss": "^3.18.0"
   },
   "scripts": {
-    "format": "npx prettier  --write '**/*.{html,js,json,md,json,yaml,yml,jsx}'",
-    "format:check": "npx prettier  --check '**/*.{html,js,json,md,json,yaml,yml}'",
-    "lint": "npx stylelint '**/*.css' --fix",
-    "lint:check": "npx stylelint '**/*.css'",
+    "format": "npx prettier  --write **/*.{html,js,json,md,json,yaml,yml,jsx}",
+    "format:check": "npx prettier  --check **/*.{html,js,json,md,json,yaml,yml}",
+    "lint": "npx stylelint **/*.css --fix",
+    "lint:check": "npx stylelint **/*.css",
```

Comment:
Is there a way you can revert this. Is this hiding so many linter issues?

Reply:
Sure,thanks

Reply:
It seems to throw an error when changes were revert
![Screenshot (123)](https://user-images.githubusercontent.com/66112760/106446081-ae488f80-6434-11eb-8bc3-d4fa8be9b4c0.png)

Reply:
You can use `"` instead but escape it, e.g. 

```js
    "format": "npx prettier  --write \"**/*.{html,js,json,md,json,yaml,yml,jsx}\"",
    "format:check": "npx prettier  --check \"**/*.{html,js,json,md,json,yaml,yml}\"",
    "lint": "npx stylelint \"**/*.css\" --fix",
    "lint:check": "npx stylelint \"**/*.css\""
```